### DISCUSSION TITLE
Next Steps/Erweiterungen?

### DISCUSSION THREAD

**Discussion by nicohuelshorst** [frickelzeugs/FingerprintDoorbell#49] [Q&A]
Hey, 
ersteinaml großen Respekt für diese Innovative Klingel und vielen Dank, dass Du diese mit uns teilst. 
Hab großes Interesse mir diese ebenfalls nachzubauen und einzusetzen.

Um jedoch den vollen Umfang einer modernen Klingel (ala Blink) nicht außen vorzulassen, stellt sich mir die Frage, ob Du hier auch schon weitere Features wie z.B. Gegensprechanlage, bzw. Videofeed geplant hast? 

Gruß

**Comment by mkrasselt1** [comment]
Das überschreitet die Leistungsfähigkeit eines Microcontrollers etwas - zumindest wenn du einen Funktionsumfang wie eine Klassiche Videoklingel erwartest - ein paar Standbilder konnte man mit einer esp-cam machen, 
aber in echtzeit, vllt. sogar zwei wege Kommunikation liegt außerhalb der Fähigkeiten des ESP.
Selbst die ESP-Audio Kits, können nicht gleichzeitig Ton abspielen und aufnehmen ... Leider bleibt dir da nur eine Wechsel auf einen Raspberry PI - Zero ? mit Kamera :)
